src/docs/services/devservices.mdx

```diff
@@ -43,15 +43,22 @@ docker exec -it sentry_postgres psql -U postgres
 
 ## Removing container state
 
-Should you really bamboozle your containers or volumes, you can use docker to remove them. As
-an example lets say we've managed to corrupt our postgres database while working on a migration,
-and we want to start over:
+Should you really bamboozle your containers or volumes, you can use `devservices rm` to start over.
 
 ```shell
 # First ensure you containers are off
```

Comment:
typo with `you`, though it can be deleted entirely; `down` is also unnecessary if you're going to `rm` everything, just `rm` is safe to do.